packages/@uppy/dashboard/src/components/AddFiles.js

```diff
@@ -1,101 +1,113 @@
-const ActionBrowseTagline = require('./ActionBrowseTagline')
 const { localIcon } = require('./icons')
 const { h, Component } = require('preact')
 
-const poweredByUppy = (props) => {
-  return <a tabindex="-1" href="https://uppy.io" rel="noreferrer noopener" target="_blank" class="uppy-Dashboard-poweredBy">{props.i18n('poweredBy')} <svg aria-hidden="true" class="UppyIcon uppy-Dashboard-poweredByIcon" width="11" height="11" viewBox="0 0 11 11">
-    <path d="M7.365 10.5l-.01-4.045h2.612L5.5.806l-4.467 5.65h2.604l.01 4.044h3.718z" fill-rule="evenodd" />
-  </svg><span class="uppy-Dashboard-poweredByUppy">Uppy</span></a>
-}
 class AddFiles extends Component {
   constructor (props) {
     super(props)
-    this.handleClick = this.handleClick.bind(this)
+
+    this.triggerFileInputClick = this.triggerFileInputClick.bind(this)
+
+    this.renderPoweredByUppy = this.renderPoweredByUppy.bind(this)
+    this.renderHiddenFileInput = this.renderHiddenFileInput.bind(this)
+    this.renderDropPasteBrowseTagline = this.renderDropPasteBrowseTagline.bind(this)
+    this.renderMyDeviceAcquirer = this.renderMyDeviceAcquirer.bind(this)
+    this.renderAcquirer = this.renderAcquirer.bind(this)
   }
 
-  handleClick (ev) {
-    this.input.click()
+  triggerFileInputClick () {
+    this.fileInput.click()
   }
 
-  render () {
-    const hasAcquirers = this.props.acquirers.length !== 0
+  renderPoweredByUppy () {
+    return <a tabindex="-1" href="https://uppy.io" rel="noreferrer noopener" target="_blank" class="uppy-Dashboard-poweredBy">
+      {this.props.i18n('poweredBy')}
+      <svg aria-hidden="true" class="UppyIcon uppy-Dashboard-poweredByIcon" width="11" height="11" viewBox="0 0 11 11">
```

Comment:
Spacing between “powered by” text and “Uppy” with icon is gone, adding something like `{' '}` should fix it, I think.

Reply:
:-o Yes